src/app/app.component.ts

```diff
@@ -1,10 +1,66 @@
-import { Component } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
+import {GitHubUsersService} from './git-hub-users.service';
+import {GitHubUsers} from './git-hub-users';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
-  title = 'angular-training';
+export class AppComponent implements OnInit{
+  // title of the component
+  public title = 'angular-training';
+
+  // store of github users
+  public gitHubUsers = '';
+  private gitHubUsersHolder: Array<GitHubUsers>;
+
+  // the number of users to present
+  N_USERS = 5;
+
+  // constructor populating private variables
+  constructor(private gitHubUserService: GitHubUsersService) {}
+
+  /**
+   * converts the array of git hub users into a strong for presenting
+   * @private
+   */
+  private convertToString(): void {
+      // go over the users and convert to the string
+      const tabSpace = '\t\t\t';
+      this.gitHubUsers += (
+          'user id' + tabSpace + 'user login' + tabSpace +
```

Comment:
The exercise description suggested using _template literals_ (template strings) as an alternative to string concatenation for composing table lines. Both options do the job, of course; the exercise was meant as a way to learn that new construct. (Possibly you knew templates already, in which case please ignore this comment.)